portal-kernel/src/com/liferay/exportimport/kernel/lar/PortletDataHandler.java

```diff
@@ -179,6 +179,48 @@ public default String getNamespace() {
 
 	public int getRank();
 
+	/**
+	 * Returns the schema version of the <code>PortletDataHandler</code> and
+	 * generally the whole staging and export/import aspect of a component. An
```

Comment:
@matethurzo Would the component we're talking about here be the one this data handler is contained within? For example, the `BookmarksPortletDataHandler` would reside in the `Bookmarks` component. Is that what we're talking about here?